CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ add_library(SerialDevice
   external/SerialDevice/src/SerialDevice.cc
 )
 
-add_executable(s3000_laser src/s3000_laser.cc src/sicks3000.cc)
+add_executable(s3000_laser src/s3000_node.cc src/sicks3000.cc)
```

Comment:
Hopefully this clears up some of the confusion between these two files.  I'm not sure about you guys, but this drove me nuts.